mobile/ios/ios-emulator-services.ts

```diff
@@ -93,3 +106,5 @@ class IosEmulatorServices implements Mobile.IEmulatorPlatformServices {
 	}
 }
 $injector.register("iOSEmulatorServices", IosEmulatorServices);
+
+export = IosEmulatorServices;
```

Comment:
Wouldn't this be equivalent to `module.exports.IosEmulatorService = IosEmulatorService`, whereas I need it to be `module.exports = IosEmulatorService`?